TestPlaybooks/playbook-playbook-CymulateTest.yml

```diff
@@ -0,0 +1,380 @@
+id: playbook-CymulateTest
```

Comment:
change the name to be just `playbook-CymulateTest.yml`
The name of the test to be `CymulateTest`